survey-runner-application/kms.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_kms_key" "credstash" {
+  description             = "KMS key for CredStash"
```

Comment:
should a `tag` be added for the environment name (can't remember how we do this elsewhere)?